scripts/tools/initialize_virtualenv.py

```diff
@@ -21,11 +21,17 @@
 import sys
 from pathlib import Path
 
+if __name__ not in ("__main__", "__mp_main__"):
+    raise SystemExit(
+        "This file is intended to be executed as an executable program. You cannot use it as a module."
+        f"To run this script, run the ./{__file__} command"
+    )
+
 
-def clean_up_airflow_sources(airflow_sources: Path):
-    if airflow_sources.exists():
-        print(f"Removing {airflow_sources}")
-        shutil.rmtree(airflow_sources, ignore_errors=True)
+def clean_up_airflow_home(airflow_home: Path):
+    if airflow_home.exists():
+        print(f"Removing {airflow_home}")
+        shutil.rmtree(airflow_home, ignore_errors=True)
 
 
 def check_if_in_virtualenv() -> bool:
```

Comment:
Few lines below there is the "devel-all" extra but it is actually wrong - it should be "devel_all" instead.